programs/cli_wallet/main.cpp

```diff
@@ -157,7 +178,8 @@ int main( int argc, char** argv )
          return 0;
       }
 
-      setup_logging();
+      setup_logging(options.at("logs-rpc-console-level").as<string>(),options.at("logs-rpc-file").as<bool>(),
+            options.at("logs-rpc-file-level").as<string>(), options.at("logs-rpc-file-name").as<string>());
```

Comment:
Will throw exceptions if `options` doesn't contain one of the options? I guess default values are needed.

Reply:
The default values are in the `main` function, where command line options are added, for example:
`("logs-rpc-file", bpo::value<bool>()->default_value(true), "Turn on/off file logging")`

A simple dump just before calling `setup_logging()` starting the wallet without any of the logging options will show:

```
1518301ms th_a       main.cpp:181                  main                 ] options.at("logs-rpc-console-level").as<string>(): info 
1518301ms th_a       main.cpp:182                  main                 ] options.at("logs-rpc-file").as<bool>(): true 
1518301ms th_a       main.cpp:183                  main                 ] options.at("logs-rpc-file-level").as<string>(): debug 
1518301ms th_a       main.cpp:184                  main                 ] options.at("logs-rpc-file-name").as<string>(): rpc.log 
```

Reply:
OK